ompi/include/mpi.h.in

```diff
@@ -806,10 +791,15 @@ enum {
    protection for this case. */
 
 #if !defined(OMPI_COMPILING_FORTRAN_WRAPPERS)
+/*
+ * Deprecated Defines:
+ */
 #define MPI_NULL_DELETE_FN OMPI_C_MPI_NULL_DELETE_FN
 #define MPI_NULL_COPY_FN OMPI_C_MPI_NULL_COPY_FN
 #define MPI_DUP_FN OMPI_C_MPI_DUP_FN
 
+/***/
```

Comment:
What is this for?

Reply:
There are lines in the MPI document separating these sections of them, so I added this.  I can remove.